lib/cc/engine/duplication.rb

```diff
@@ -31,6 +32,10 @@ def run
           reporter = CC::Engine::Analyzers::Reporter.new(directory, engine, io)
           reporter.run
         end
+
+      rescue EmptyLanguagesError => ex
+        $stderr.puts ex.message
```

Comment:
What do you think of doing the `$stderr.puts ...` below (inside of `languages.empty?`)? That way you wouldn't have to rescue and re-raise